### ISSUE TITLE
On iPhone, the accessibility for slider, checkbox, and radio buttons are poor

### ISSUE THREAD

**Issue by lightboys22** [xinthink/react-native-material-kit#270]
If the user turned on voice over, ...
- the slider should announce what is the current value.
- the checkbox / radio buttons should announce if it is checked or not.
- the buttons should have a UIAccessibilityTraitButton trait.

**Comment by alindj** [comment]
Did you find any solutions for this
